src/azul/service/manifest_service.py

```diff
@@ -1005,8 +1005,8 @@ def source_filter(self) -> SourceFilters:
     @classmethod
     def command_lines(cls, url: str) -> JSON:
         return {
-            'cmd.exe': f'curl.exe --location {cls._cmd_exe_quote(url)} | curl.exe --config -',
-            'bash': f'curl --location {shlex.quote(url)} | curl --config -'
+            'cmd.exe': f'curl.exe --location -f {cls._cmd_exe_quote(url)} | curl.exe --config -',
+            'bash': f'curl --location -f {shlex.quote(url)} | curl --config -'
```

Comment:
This only prevents a file from being written with the contentes of the 4xx response. Is this sufficient?

I wasn't able to find a way to direct the message/body of the failing response to the `stdout/stderr` of the curl command.
![Screen Shot 2021-07-12 at 5 37 17 PM](https://user-images.githubusercontent.com/19215510/125373112-627e0880-e339-11eb-9b57-7eaee9a070de.png)

Reply:
It's nice you included this example of what the actual error looks like. It's a bummer we can't get the whole response in there, but I think this is definitely less obscure than before.